pytket/extensions/qiskit/backends/aer.py

```diff
@@ -134,9 +127,74 @@ def backend_info(self) -> BackendInfo:
 
     def rebase_pass(self) -> BasePass:
         return auto_rebase_pass(
-            self._gate_set,
+            self._backend_info.gate_set,
         )
 
+    def _arch_dependent_default_compilation_pass(
+        self, arch: Architecture, optimisation_level: int = 2
+    ) -> BasePass:
+        assert optimisation_level in range(3)
+        arch_specific_passes = [
+            CXMappingPass(
+                arch,
+                NoiseAwarePlacement(
```

Comment:
Do we always want to do noise aware placement for all optimisation_level?

Reply:
I don't know. This is how it was before